### ISSUE TITLE
Validation loss and accuracy remains constant

### ISSUE THREAD

**Issue by arminbiglari** [qiangyao1988/IA-ViT#1]
Hello, Thank you for your great work.
I am training the model on my own dataset. but during tranining my validation accuracy remains constant at 0.80 and my validation loss fluctuate around 0.5, do you know what is the problem?

i appreciate your time and consideration.
looking forward for your response.
